bude_hezky/main.py

```diff
@@ -0,0 +1,36 @@
+import argparse, os
+from dotenv import load_dotenv
+from bude_hezky.weather import weather_forecast, weather_resolver
+from bude_hezky.sender import email_sender
+
+CITY_KEY = 'mesto'
+EMAIL_KEY = 'email'
+CHAT_ID = 'chat'
+FROM_EMAIL = 'ivan@ikvasnica.com'
+
+parser = argparse.ArgumentParser()
+parser.add_argument(CITY_KEY, help='Město, kde bydlíš. Můžeš zkusit i vesnici.')
+parser.add_argument('--' + EMAIL_KEY, help='E-mail, na který pošleme informaci, zda bude hezky.')
```

Comment:
V nových Pythonech můžeš udělat i `f'--{EMAIL_KEY}'`, pokud by se ti to líbilo víc.

Reply:
😍